src/Video.js

```diff
@@ -161,8 +164,7 @@ export default class Video extends React.Component {
     playPressedFirstTime = true;
     secondsPlayed = 0;
     clearTimeout(this.controlsTO);
-    clearTimeout(this.bufferingTO);
-    clearTimeout(this.bufferingTooLongTO);
+    // clearTimeout(this.bufferingTooLongTO);
```

Comment:
Left some comments 👀

Reply:
I know I was waiting for Joel to tell me if we need that feature or not :D will remove it

Reply:
Makes sense.